cmd/lambda/usage/list.go

```diff
@@ -35,7 +35,15 @@ func GetUsage(w http.ResponseWriter, r *http.Request) {
 	// Serialize them for the JSON response.
 	usageResponseItems := []response.UsageResponse{}
 	for _, usageItem := range result.Results {
-		usageResponseItems = append(usageResponseItems, response.UsageResponse(*usageItem))
+		usageResponseItems = append(usageResponseItems, response.UsageResponse{
```

Comment:
Do we need a `response.UsageResponse` struct? Or can we just use the `usage.Usage` model struct?

Reply:
Yea @eschwartz Going to work on that next.  Little bit different of a change here because pkg/usage already existed so I needed to rework some stuff first.